src/dbus.c

```diff
@@ -694,6 +694,76 @@ static struct notification *dbus_message_to_notification(const gchar *sender, GV
         return n;
 }
 
+void signal_length_propertieschanged()
+{
+
+        static unsigned int last_displayed = 0;
+        static unsigned int last_history = 0;
+        static unsigned int last_waiting = 0;
+
+        if (!dbus_conn)
+                return;
+
+        GVariantBuilder *builder = g_variant_builder_new(G_VARIANT_TYPE_VARDICT);
+        GVariantBuilder *invalidated_builder = g_variant_builder_new(G_VARIANT_TYPE_STRING_ARRAY);
+
+        unsigned int displayed = queues_length_displayed();
+        unsigned int history = queues_length_history();
+        unsigned int waiting = queues_length_waiting();
+        bool properties_changed = false;
+
+        if (last_displayed != displayed)
+        {
+                g_variant_builder_add(builder,
+                                      "{sv}",
+                                      "displayedLength", g_variant_new_uint32(displayed));
+                last_displayed = displayed;
+                properties_changed = true;
+        }
+
+        if (last_history != history)
```

Comment:
Please use consistent style for this. IIRC it should be this:

```suggestion
        if (last_history != history){
```

Reply:
It seems like there are a handful of inconsistent formatting (extra line break or no space between the `)` and `{`).
I'll fix them in a separate commit.